### ISSUE TITLE
Cannot install Pale Moon and Falcon

### ISSUE THREAD

**Issue by touhidurrr** [Botspot/pi-apps#1422]
### Confirmations

- [X] I searched for similar bug reports in https://github.com/Botspot/pi-apps/issues and found none were relevant.
- [X] I tried updating Pi Apps and the issue is still not fixed. [How to update Pi Apps?](https://github.com/Botspot/pi-apps#to-update-pi-apps)

### What happened?

Unable to Install the apps

### Description

Tried several times but cannot install them

### What is your operating system?

Raspberry Pi OS armhf, bullseye

### (Optional) Error log? Terminal output? Debug messages?

```bash
Installing Pale Moon...

Will install these packages: palemoon 
Running sudo apt update...
Hit:1 http://archive.raspberrypi.org/debian bullseye InRelease
Hit:2 https://itai-nelken.github.io/weekly-box86-debs/debian  InRelease
Hit:4 https://deb.nodesource.com/node_17.x bullseye InRelease
Hit:5 http://raspbian.raspberrypi.org/raspbian bullseye InRelease
Hit:6 http://ports.ubuntu.com/ubuntu-ports bionic-updates InRelease
Ign:7 https://download.puffinbrowser.com/repo stable InRelease
Ign:8 https://repo.vivaldi.com/archive/deb stable InRelease
Hit:10 https://download.puffinbrowser.com/repo stable Release
Hit:11 https://adoptium.jfrog.io/artifactory/deb bullseye InRelease
Ign:12 https://repo.vivaldi.com/stable/deb stable InRelease
Hit:13 https://repo.vivaldi.com/archive/deb stable Release
Hit:14 https://repo.vivaldi.com/stable/deb stable Release
2 packages can be upgraded. Run 'apt list --upgradable' to see them.
apt update complete.
Some packages can be upgraded. Please consider running sudo apt full-upgrade.
Creating an empty apt-package to install the necessary apt packages...
It will be named: pi-apps-338c040f
Depends: palemoon
Installing the pi-apps-338c040f package...
Waiting until APT locks are released... Done
The following additional packages will be installed:
  palemoon
The following NEW packages will be installed:
  palemoon pi-apps-338c040f
0 upgraded, 2 newly installed, 0 to remove and 0 not upgraded.
Err:2 http://download.opensuse.org/repositories/home:/stevenpusser/Raspbian_10  palemoon 29.4.4-1.gtk2
  Error reading from server. Remote end closed connection [IP: 195.135.221.134 80]
E: Failed to fetch http://download.opensuse.org/repositories/home:/stevenpusser/Raspbian_10/./armhf/palemoon_29.4.4-1.gtk2_armhf.deb  Error reading from server. Remote end closed connection [IP: 195.135.221.134 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Apt finished.
Failed to install the packages!
APT reported these errors:
E: Failed to fetch http://download.opensuse.org/repositories/home:/stevenpusser/Raspbian_10/./armhf/palemoon_29.4.4-1.gtk2_armhf.deb  Error reading from server. Remote end closed connection [IP: 195.135.221.134 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
The following additional packages will be installed:
  palemoon
The following NEW packages will be installed:
  palemoon pi-apps-338c040f
0 upgraded, 2 newly installed, 0 to remove and 0 not upgraded.
Err:2 http://download.opensuse.org/repositories/home:/stevenpusser/Raspbian_10  palemoon 29.4.4-1.gtk2
  Error reading from server. Remote end closed connection [IP: 195.135.221.134 80]
E: Failed to fetch http://download.opensuse.org/repositories/home:/stevenpusser/Raspbian_10/./armhf/palemoon_29.4.4-1.gtk2_armhf.deb  Error reading from server. Remote end closed connection [IP: 195.135.221.134 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?

Failed to install Pale Moon!
◢◣Need help? Copy the ENTIRE terminal output or take a screenshot.
Please ask on Github: https://github.com/Botspot/pi-apps/issues/new/choose
Or on Discord: https://discord.gg/RXSTvaUvuu
Installing Falkon...

Will install these packages: http://ftp.debian.org/debian/pool/main/q/qtwebengine-opensource-src/libqt5webenginecore5_5.11.3+dfsg-2+deb10u1_armhf.deb http://ftp.debian.org/debian/pool/main/q/qtwebengine-opensource-src/libqt5webenginewidgets5_5.11.3+dfsg-2+deb10u1_armhf.deb http://ftp.debian.org/debian/pool/main/f/falkon/falkon_3.0.0-3_armhf.deb 
Downloading libqt5webenginecore5_5.11.3+dfsg-2+deb10u1_armhf.deb... 
24MiB/24MiB ————————————————————————————————————————————————————————————————————
Done
Downloading libqt5webenginewidgets5_5.11.3+dfsg-2+deb10u1_armhf.deb... 
64KiB/123KiB ———————————————————————————————————
Done
Downloading falkon_3.0.0-3_armhf.deb... 
1.7MiB/1.8MiB ————————————————————————————————————————————————————————————
Done
Running sudo apt update...
Ign:1 file:/tmp/pi-apps-local-packages ./ InRelease
Ign:2 file:/tmp/pi-apps-local-packages ./ Release
Ign:3 file:/tmp/pi-apps-local-packages ./ Packages
Ign:4 file:/tmp/pi-apps-local-packages ./ Translation-en_US.UTF-8
Ign:5 file:/tmp/pi-apps-local-packages ./ Translation-en
Ign:6 file:/tmp/pi-apps-local-packages ./ Translation-en_US
Ign:3 file:/tmp/pi-apps-local-packages ./ Packages
Ign:4 file:/tmp/pi-apps-local-packages ./ Translation-en_US.UTF-8
Ign:5 file:/tmp/pi-apps-local-packages ./ Translation-en
Ign:6 file:/tmp/pi-apps-local-packages ./ Translation-en_US
Ign:3 file:/tmp/pi-apps-local-packages ./ Packages
Ign:4 file:/tmp/pi-apps-local-packages ./ Translation-en_US.UTF-8
Ign:5 file:/tmp/pi-apps-local-packages ./ Translation-en
Ign:6 file:/tmp/pi-apps-local-packages ./ Translation-en_US
Ign:3 file:/tmp/pi-apps-local-packages ./ Packages
Ign:4 file:/tmp/pi-apps-local-packages ./ Translation-en_US.UTF-8
Ign:5 file:/tmp/pi-apps-local-packages ./ Translation-en
Ign:6 file:/tmp/pi-apps-local-packages ./ Translation-en_US
Ign:3 file:/tmp/pi-apps-local-packages ./ Packages
Ign:4 file:/tmp/pi-apps-local-packages ./ Translation-en_US.UTF-8
Ign:5 file:/tmp/pi-apps-local-packages ./ Translation-en
Ign:6 file:/tmp/pi-apps-local-packages ./ Translation-en_US
Ign:3 file:/tmp/pi-apps-local-packages ./ Packages
Ign:4 file:/tmp/pi-apps-local-packages ./ Translation-en_US.UTF-8
Ign:5 file:/tmp/pi-apps-local-packages ./ Translation-en
Ign:6 file:/tmp/pi-apps-local-packages ./ Translation-en_US
Ign:4 file:/tmp/pi-apps-local-packages ./ Translation-en_US.UTF-8
Ign:5 file:/tmp/pi-apps-local-packages ./ Translation-en
Ign:6 file:/tmp/pi-apps-local-packages ./ Translation-en_US
Hit:7 http://archive.raspberrypi.org/debian bullseye InRelease
Hit:8 http://ports.ubuntu.com/ubuntu-ports bionic-updates InRelease
Hit:9 https://itai-nelken.github.io/weekly-box86-debs/debian  InRelease
Ign:10 https://download.puffinbrowser.com/repo stable InRelease
Hit:12 http://raspbian.raspberrypi.org/raspbian bullseye InRelease
Hit:13 https://download.puffinbrowser.com/repo stable Release
Ign:14 https://repo.vivaldi.com/archive/deb stable InRelease
Hit:15 https://deb.nodesource.com/node_17.x bullseye InRelease
Ign:16 https://repo.vivaldi.com/stable/deb stable InRelease
Hit:17 https://adoptium.jfrog.io/artifactory/deb bullseye InRelease
Hit:18 https://repo.vivaldi.com/archive/deb stable Release
Hit:19 https://repo.vivaldi.com/stable/deb stable Release
All packages are up to date.
apt update complete.
Creating an empty apt-package to install the necessary apt packages...
It will be named: pi-apps-c940c8ee
Depends: falkon, libqt5webenginecore5, libqt5webenginewidgets5
Installing the pi-apps-c940c8ee package...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:
The following packages have unmet dependencies:
 libqt5webenginecore5 : Depends: libqt5webengine-data (= 5.11.3+dfsg-2+deb10u1) but it is not going to be installed
                        Depends: libevent-2.1-6 (>= 2.1.8-stable) but it is not installable
                        Depends: libre2-5 (>= 20160901) but it is not installable
                        Depends: qtbase-abi-5-11-3 but it is not installable
 libqt5webenginewidgets5 : Depends: libqt5webengine-data (= 5.11.3+dfsg-2+deb10u1) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
Apt finished.
Failed to install the packages!
APT reported these errors:
E: Unable to correct problems, you have held broken packages.
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:
The following packages have unmet dependencies:
 libqt5webenginecore5 : Depends: libqt5webengine-data (= 5.11.3+dfsg-2+deb10u1) but it is not going to be installed
                        Depends: libevent-2.1-6 (>= 2.1.8-stable) but it is not installable
                        Depends: libre2-5 (>= 20160901) but it is not installable
                        Depends: qtbase-abi-5-11-3 but it is not installable
 libqt5webenginewidgets5 : Depends: libqt5webengine-data (= 5.11.3+dfsg-2+deb10u1) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.

Failed to install Falkon!
◢◣Need help? Copy the ENTIRE terminal output or take a screenshot.
Please ask on Github: https://github.com/Botspot/pi-apps/issues/new/choose
Or on Discord: https://discord.gg/RXSTvaUvuu

Close this window to exit.
```

**Comment by touhidurrr** [comment]
It does seem like a bug to me. broken dependencies? why?

**Comment by cycool29** [comment]
For Pale Moon, I can download [the link](http://download.opensuse.org/repositories/home:/stevenpusser/Raspbian_10/./armhf/palemoon_29.4.4-1.gtk2_armhf.deb) that your log mentions `Error reading from server. Remote end closed connection [IP: 195.135.221.134 80]`. Maybe you have a firewall or something blocking it?

For Falkon, it is currently unsupported on bullseye.

**Comment by touhidurrr** [comment]
> ```shell
> Err:2 http://download.opensuse.org/repositories/home:/stevenpusser/Raspbian_10  palemoon 29.4.4-1.gtk2
>   Error reading from server. Remote end closed connection [IP: 195.135.221.134 80]
> ```

you mean the following lines?
```bash
Err:2 http://download.opensuse.org/repositories/home:/stevenpusser/Raspbian_10  palemoon 29.4.4-1.gtk2
  Error reading from server. Remote end closed connection [IP: 195.135.221.134 80]
```
Its basically a download failure.

But it still doesn't explain me failing several times.

**Comment by Botspot** [comment]
With commit https://github.com/Botspot/pi-apps/commit/49b049f0382ac202ce5a34937bd24ea621d8a3ad, Falkin, FreeCAD, and LinuxCNC were removed from Pi-Apps to sully support Raspberry Pi OS Bullseye.
